src/Storefront/Resources/views/storefront/page/checkout/confirm/confirm-address.html.twig

```diff
@@ -45,7 +45,9 @@
                                 {% block page_checkout_confirm_address_shipping_actions %}
                                     {% set addressManagerOptions = {
                                         initialTab: 'shipping',
-                                        activeShippingAddressId: shippingAddress.id
+                                        activeShippingAddressId: shippingAddress.id,
+                                        addressManagerUrl: url('frontend.account.addressmanager.get'),
+                                        addressSwitchUrl: url('frontend.account.address.switch-default'),
```

Comment:
I think `url()` will generate based on the `APP_URL` which may not necessarily match the sales channel and could cause CORS errors:
```suggestion
                                        addressManagerUrl: path('frontend.account.addressmanager.get'),
                                        addressSwitchUrl: path('frontend.account.address.switch-default'),
```

Reply:
Untrue, it also generates with the Sales Channel Domain, but absolute and not relative. So I will use path.